installing/installing_gcp/installing-gcp-private.adoc

```diff
@@ -0,0 +1,53 @@
+[id="installing-gcp-private"]
+= Installing a private cluster on GCP
+include::modules/common-attributes.adoc[]
+:context: installing-gcp-private
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a private cluster into an existing VPC on Google Cloud Platform (GCP). The installation program provisions the rest of the required infrastructure, which you can further customize. To customize the installation, you modify
+parameters in the `install-config.yaml` file before you install the cluster.
+
+.Prerequisites
+
```

Comment:
If your workstation on which installation program runs is outside of  VPC network, you cannot validate the cluster status cause the cluster has only internal access. It's better to create a bastion host within the VPC network and run installation program on it.

Reply:
@shellyyang1989, @jianlinliu, do you have information about the bastion host that's required?

Reply:
I think some PR about bastion host already in progress:
https://github.com/openshift/openshift-docs/pull/16989/files

Reply:
I had a chat with Jeremiah, and he thinks that the bastion host isn't necessarily the way to go. We agreed that it's going to be better to talk about the access that he host that you install the private cluster on needs instead of proscribing a bastion, at least for now. Jeremiah used a bastion but thinks that a VPN or other existing access will be more likely.